pyproject.toml

```
@@ -43,10 +43,10 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "<3.12,>=3.7"
-structlog = "^21.2.0"
-PyYAML = "^6.0.0"
+structlog = "^21"
+PyYAML = ">=6.0.0,<7"
```

`"^6.0.0"` is equivalent to `">=6.0.0,<7"`

```suggestion
PyYAML = "^6.0.0"
```

Reference: https://python-poetry.org/docs/dependency-specification/#caret-requirements